coffee/plan.py

```diff
@@ -138,12 +138,24 @@ def plan_cpu(self, opts):
                         # of the expression
                         vect.specialize(*vectorize)
 
+            # First we check if our kernel function is templated
+            templated = False
+            for q in kernel.pred:
+                if q == 'template <typename Derived>':
+                    templated = True
+
```

Comment:
more pythonic:
``templated = any(q == 'template <typename Derived>' for q in kernel.pred)``